src/render/program/fill_extrusion_program.js

```diff
@@ -80,7 +84,8 @@ const extrusionTextureUniforms = (context: Context, locations: UniformLocations)
 
 const fillExtrusionUniformValues = (
     matrix: Float32Array,
-    painter: Painter
+    painter: Painter,
+    verticalGradient: number
```

Comment:
For clarity, I lean towards passing this value to the `*UniformValues` functions as a boolean, and casting only as the value is bound to its uniform location.

Reply:
`Uniform1i` requires a number input so casting the boolean to a number only when binding to the location causes a Flow error. Is there a way around that?

Reply:
yep you use the `+` like here https://github.com/mapbox/mapbox-gl-js/blob/master/src/render/program/symbol_program.js#L116

Reply:
Ah yeah, I was mistakenly casting to a number elsewhere in the program, causing a flow error. This is fixed now.